privacyidea/config.py

```diff
@@ -33,10 +34,10 @@
 
 def _random_password(size):
     log.info("SECRET_KEY not set in config. Generating a random key.")
-    passwd = [random.choice(string.ascii_lowercase + \
+    passwd = [secrets.choice(string.ascii_lowercase + \
                             string.ascii_uppercase + string.digits) for _x in range(size)]
-    # return shuffled password
-    random.shuffle(passwd)
+    # return shuffled password - as the password was already randomly generated, this might actually be not necessary
+    random.shuffle(passwd)  #
```

Comment:
I vote for removing the `shuffle` here. That way we can get rid of the `random` import as well.